addon/components/base/bs-dropdown/button.js

```diff
@@ -15,7 +15,7 @@ import { schedule } from '@ember/runloop';
 @classNames('dropdown-toggle')
 export default class DropdownButton extends Button {
   didReceiveAttrs() {
-    super.didReceiveAttrs(...arguments);
+    // super.didReceiveAttrs(...arguments);
```

Comment:
`didReceiveAttrs()` is missing a noop implementation on `@ember/component`, isn't it?

Reply:
IIRC this was the problem, yes. `this._super` seems to always have a noop implementation.